src/Illuminate/Queue/QueueManager.php

```diff
@@ -233,6 +234,16 @@ public function setDefaultDriver($name)
     }
 
     /**
+     * Get the name of the queue prefix.
+     *
+     * @return string
+     */
+    public function getQueuePrefix()
+    {
+        return isset($this->app['config']['queue.prefix']) ? $this->app['config']['queue.prefix'] : null;
```

Comment:
`$this->app['config']->get(''queue.prefix')` and `$this->app['config']['queue.prefix']` is have identical behavior. And there no need to `isset` check, because if this key (`'queue.prefix'`) is not exists then default value (`null`) will be returned.